applications/machine_learning/README.rst

```diff
@@ -275,6 +275,28 @@ By default, the application uses the following LED effects:
     * LED blinks with an average frequency if it is connected, but is not actively forwarding data.
     * LED blinks rapidly if it is connected and is actively forwarding data.
 
+Power management
+================
+
+Reducing power consumption is important for every battery-powered device.
+
+In the Machine Learning application specified modules can be suspended or power off there are not in use.
+
+* In the suspended state, the device maintains the active connection.
```

Comment:
`the active Bluetooth connection`?
To distinguish from data forwarding over EI data forwarder protocol (not every configuration supports Bluetooth in case of the ML application)

Reply:
done